units/en/unitbonus1/train.mdx

```diff
@@ -0,0 +1,265 @@
+
+```python
+%%capture
+!unzip -d ./trained-envs-executables/linux/ ./trained-envs-executables/linux/Huggy.zip
+```
+
+Make sure your file is accessible
+
+```python
+!chmod -R 755 ./trained-envs-executables/linux/Huggy
+```
+
+## Let's recap how this environment works
+
+### The State Space: what Huggy "perceives."
+
+Huggy doesn't "see" his environment. Instead, we provide him information about the environment:
+
+- The target (stick) position
+- The relative position between himself and the target
+- The orientation of his legs.
+
+Given all this information, Huggy **can decide which action to take next to fulfill his goal**.
+
+<img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/notebooks/unit-bonus1/huggy.jpg" alt="Huggy" width="100%">
+
+
+### The Action Space: what moves Huggy can do
+<img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/notebooks/unit-bonus1/huggy-action.jpg" alt="Huggy action" width="100%">
+
+**Joint motors drive huggy legs**. It means that to get the target, Huggy needs to **learn to rotate the joint motors of each of his legs correctly so he can move**.
+
+### The Reward Function
+
+The reward function is designed so that **Huggy will fulfill his goal** : fetch the stick.
+
+Remember that one of the foundations of Reinforcement Learning is the *reward hypothesis*: a goal can be described as the **maximization of the expected cumulative reward**.
+
+Here, our goal is that Huggy **goes towards the stick but without spinning too much**. Hence, our reward function must translate this goal.
+
+Our reward function:
+
+<img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/notebooks/unit-bonus1/reward.jpg" alt="Huggy reward function" width="100%">
+
+- *Orientation bonus*: we **reward him for getting close to the target**.
+- *Time penalty*: a fixed-time penalty given at every action to **force him to get to the stick as fast as possible**.
+- *Rotation penalty*: we penalize Huggy if **he spins too much and turns too quickly**.
+- *Getting to the target reward*: we reward Huggy for **reaching the target**.
+
+## Check  the Huggy config file
+
+- In ML-Agents, you define the **training hyperparameters into config.yaml files.**
+
+- For the scope of this notebook, we're not going to modify the hyperparameters but if you want to try as an experimentation, you should also try to modify some other hyperparameters, Unity provides a very [good documentation explaining each of them here](https://github.com/Unity-Technologies/ml-agents/blob/main/docs/Training-Configuration-File.md).
+
+- Click here to open the config.yaml: `/content/ml-agents/config/ppo/Huggy.yaml`
```

Comment:
Does this work in Colab? What do they click?

Reply:
Yes it works I added "In google colab"